CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+### 0.1.5
+
+ * Added API function `authorization-check` to the AuthorizationService
+   protocol to support authorization from non-Ring-based handlers such as Java
+   Servlets.
+ * [TK-268](https://tickets.puppetlabs.com/browse/TK-268) Weight 'deny' ACEs
```

Comment:
Would be good to mention TK-265 here, too - specifically that we removed support for IP matching.